.travis.yml

```diff
@@ -1,7 +1,6 @@
-env:
-  global:
-  - DEBUG='* -superagent -express:* -mocha:*'
 language: node_js
+sudo: false
 node_js:
 - '0.12'
+- stable
```

Comment:
Stable so we also get recent node (I suspect we won't be able to keep up with coming releases so just adding stable seems sane).